recipes/gdal/post_3.5.0/conandata.yml

```diff
@@ -12,9 +12,15 @@ patches:
   "3.7.0":
     - patch_file: "patches/3.7.0/0-replace-find-package.patch"
       base_path: "source_subfolder"
+    - patch_file: "patches/3.7.0/1-python-binding-find-libdir.patch"
+      base_path: "source_subfolder"
```

Comment:
Could you add a patch_description info to these patches, please?

Reply:
Sure, done.